adhocracy4/projects/models.py

```diff
@@ -152,6 +152,14 @@ def get_current_modules(self, idx):
             return []
         return []
 
+    def get_module_cluster_index(self, module):
+        for idx, val in enumerate(self.participation_dates):
+            if "modules" in val and module in val["modules"]:
+                return idx
+        if self.project.get_current_participation_date():
+            return self.project.get_current_participation_date()
+        return 0
+
```

Comment:
Is in that order bcz participation_dates may both exist? so we prioritise the modules over the project's?

Reply:
So the order is because first, I try to find the module cluster that the module is in, if that doesnt work, I take the module cluster that is the 'active' one (this is what get_current_participation_date() does, I copied it from initial_slide() in DisplayProjectOrModuleMixin). It is more like a fallback.